JS/Closures/example1.html

```diff
@@ -2,17 +2,17 @@
 <html>
 <body>
 
-<p>A function  displayName() can access variables defined inside the function me() :</p>
+<p>A function  displayName() can access variables defined inside the function pam() :</p>
 
-<!-- <button type="button" onclick="me()">Click Me!</button> -->
+<!-- <button type="button" onclick="pam()">Click Me!</button> -->
 <!-- <p id="demo2"></p> -->
 <p id="demo"></p>
 
 
 <script>
```

Comment:
Always create a new file.
Move the below code to a new file and link it on this page.